lnrpc/rpc.proto

```diff
@@ -717,6 +717,8 @@ message SendRequest {
     send the payment.
     */
     FeeLimit fee_limit = 8;
+
+    int32 spiderAlgo = 9;
```

Comment:
Can you please add a comment here like the above fields explaining what this parameter is?